### DISCUSSION TITLE
storage/solutions/helm-rook-ceph/

### DISCUSSION THREAD

**Discussion by giscus** [DaoCloud/DaoCloud-docs#4659] [Announcements]
# storage/solutions/helm-rook-ceph/

DaoCloud Enterprise 5.0 下一代容器化平台集大成者，引领云原生浪潮，推动数字化转型。

https://docs.daocloud.io/storage/solutions/helm-rook-ceph/

**Comment by JiangYeZeng** [comment]
您好，此文档 并无展示toolbox.yaml文件，以至于我去github下载官网的yaml文件，但最后发现少了osd和mon 这个关键组件的pod，请问谁可以提供下，基于此文档的toolbox.yaml文件呢

**Reply by harveyjing** [reply]
https://raw.githubusercontent.com/rook/rook/v1.14.8/deploy/examples/toolbox.yaml

**Comment by windsonsea** [comment]
Fixed in PR #5282
